openapi.yaml

```diff
@@ -17,69 +17,116 @@ openapi: 3.0.3
-      operationId: postExample
+      summary: Creates a Kentix configuration
+      description: Creates a Kentix configuration.
+      operationId: postConfiguration
       requestBody:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/Example'
+              $ref: '#/components/schemas/Configuration'
       responses:
         201:
-          description: Successfully created an example configuration
+          description: Successfully created a new Kentix configuration
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Example'
+                $ref: '#/components/schemas/Configuration'
 
 components:
   schemas:
-
-    Example:
+    Configuration:
       type: object
-      description: Simply a example configuration
+      description: Each configuration defines access to one Kentix device.
       properties:
         id:
           type: integer
           format: int64
-          description: A id identifying the example configuration
+          description: Internal identifier for the configured device (created automatically). This identifier have to use always if you remove or update existing configured endpoints.
+          readOnly: true
+          nullable: true
+        address:
+          type: string
+          format: string
+          description: IP or hostname of the Kentix device
+          example: 10.10.10.101
+        apiKey:
+          type: string
+          description: Kentix API key
+        enable:
+          type: boolean
+          description: Flag to enable or disable the device
+          default: true
+          nullable: true
+        refreshInterval:
+          type: integer
+          description: Interval in seconds for collecting data from device
+          default: 60
+        requestTimeout:
+          type: integer
+          description: Timeout in seconds
+          default: 120
+          nullable: true
+        active:
+          type: boolean
           readOnly: true
+          description: Set to `true` by the app when running and to `false` when app is stopped
+          nullable: true
+        projectIDs:
+          type: array
+          description: List of Eliona project ids for which this device should collect data. For each project id all smart devices are automatically created as an asset in Eliona. The mapping between Eliona is stored as an asset mapping in the Kentix app.
+          nullable: true
+          items:
+            type: string
+          example:
+            - 42
+            - 99
+
+    Sensor:
+      type: object
+      description: Each sensor represents one asset in Eliona.
+      properties:
+        projectID:
+          type: integer
+          description: The project ID this asset is assigned to.
+        assetID:
+          type: integer
+          description: ID of a parent asset which groups all device assets
```

Comment:
The assetId is not the ID of the PARENT asset. The assestId ist the ID of the asset corresponding to the sensor. Each sensor has an corresponding asset.